recipes/docutils/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{%set name = "docutils" %}
+{%set version = "0.12" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "c7db717810ab6965f66c8cf0398a98c9d8df982da39b4cd7f162911eb89596fa" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - docutils
+    - docutils.languages
+    - docutils.parsers
+    - docutils.parsers.rst
+    - docutils.parsers.rst.directives
+    - docutils.parsers.rst.languages
+    - docutils.readers
+    - docutils.transforms
+    - docutils.utils
+    - docutils.utils.math
+    - docutils.writers
+    - docutils.writers.html4css1
+    - docutils.writers.latex2e
+    - docutils.writers.odf_odt
+    - docutils.writers.pep_html
+    - docutils.writers.s5_html
+    - docutils.writers.xetex
+
+about:
+  home: http://docutils.sourceforge.net/
+  license: Public Domain Dedictation and BSD 2-Clause and BSD 3-Clause and PSF 2.1.1 and GPL 3.0
```

Comment:
I don't think there is any BSD 3-Clause in there, but could be shown to be wrong.